docs/api/benchmarks.rst

```diff
@@ -1,24 +0,0 @@
-.. currentmodule:: pythainlp.benchmarks
```

Comment:
- [ ] Add benchmarks docs

Reply:
That was my mistake, I already fixed it. 😥